selftest/percpu/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import "C"
+
+import (
+	"encoding/binary"
+	"fmt"
+	"os"
+	"runtime"
+	"syscall"
+	"time"
+	"unsafe"
+
+	bpf "github.com/aquasecurity/libbpfgo"
+)
+
+func main() {
+
+	bpfModule, err := bpf.NewModuleFromFile("main.bpf.o")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(-1)
+	}
+	defer bpfModule.Close()
+
+	err = bpfModule.BPFLoadObject()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(-1)
+	}
+
+	prog, err := bpfModule.GetProgram("mmap_fentry")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(-1)
+	}
+
+	link, err := prog.AttachGeneric()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(-1)
+	}
+	if link.GetFd() == 0 {
+		os.Exit(-1)
+	}
+
+	go func() {
+		for {
+			syscall.Mmap(999, 999, 999, 1, 1)
```

Comment:
This call is broken (like the other selftest with the same call). If you check for error here you will see this is not working (at least in the other exact same call for another PR).